.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
-ignore = F401,E711,W503
+ignore = F401,E711,W503,E203
```

Comment:
Could you comment on why this additional ignore is needed?

Reply:
E203 whitespace before ':'
This incompatible with black.

Reply:
That is surprising to me as I did not have the same problem in https://github.com/equinor/ert/pull/1452. Could you try to remove it and see whether the CI gives you trouble? If not, I suspect you have an outdated version of flake8 or black in your environment 🤷

Reply:
I found out about it via the CI, and recently reinstalled, so I figured the issue was weird, but real, and added the flag.

To be sure I checked again now:
- upgraded `black `and `flake8`: same fail
- ran CI with the flag removed: same fail

So possibly `flake8 `got updated and introduced the issue? If I recall correctly `black` never has put spaces before `:`.